### PR TITLE
Added log statment and enabled background modes for ios notifications 

### PR DESCRIPTION
**Pull request by kunall17**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

File: ios/ZulipMobile/Info.plist [hunk 1]
```diff
@@ -1,89 +1,93 @@
...
-      <string>UIInterfaceOrientationLandscapeRight</string>
-      <string>UIInterfaceOrientationPortraitUpsideDown</string>
-    </array>
-    <key>UIViewControllerBasedStatusBarAppearance</key>
-    <false/>
-  </dict>
-</plist>
\ No newline at end of file
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
```
Comment (borisyankov) [inline ios/ZulipMobile/Info.plist]: Looks good.
Can we avoid the ugly diff here?

**Comment by kunall17** [comment]
@borisyankov Done!

**Comment by borisyankov** [comment]
Looks good 👍

**Pull request merged**